components/Press/PressLinks/__tests__/PressLinks.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import { mount, shallow } from 'enzyme';
+
+import PressLinks from '../PressLinks';
+import LinkGroup from '../LinkGroup';
+
+describe('PressLinks', () => {
+  test('it should render properly no props', () => createShallowSnapshotTest(<PressLinks />));
+});
+
+describe('PressLinks > LinkGroups', () => {
```

Comment:
instead of trying to describe both within the same file, can you create a separate test file for `LinkGroup`?